Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart

```diff
@@ -0,0 +1,62 @@
+
+import 'package:dartsmiths/features/login/presentation/widgets/login_card.dart';
+import 'package:flutter/material.dart';
+
+class Login extends StatelessWidget {
+  const Login({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.white,
```

Comment:
Avoid defining colors like this. rebase the main, define the colors in the color constant file and use them.

Reply:
I used color.dart file and update the colors used in the code